DBMS/Assignment1/Assign2.sql

```diff
@@ -0,0 +1,95 @@
+
+/*Assignment Two*/
+create database PiyushSharmaDB;
+use PiyushSharmaDB;
+
+/*
+Create all tables of Library as described and mentioned in Slides. (Write all CREATE commands in a SQL file and run that SQL File).
+ */
+
+CREATE TABLE members(member_id int primary key,
+                    member_nm varchar(90),
+                    addressline1 varchar(200),
+                    addressline2 varchar(200),
+                    category varchar(3));
+                    
+CREATE TABLE book_issue(issue_dt date,
+                        accession_no int,
+                        member_id int,
+                        due_dt date,
+                        Primary Key(issue_dt,accession_no,member_id),
+                        Foreign Key(member_id) References members(member_id)
+                        );
+                        
+CREATE TABLE books(accession_no int Primary key,
+                    title_dt date,
+                    purchase_dt date,
+                    price int,
+                    status varchar(90)
+                    );
+                    
+CREATE TABLE book_return(return_dt date,
+                        accession_no int,
+                        member_id int,
+                        issue_dt date,
+                        Primary key(return_dt,accession_no,member_id),
+                        Foreign Key(accession_no) References books(accession_no),
+                        Foreign key(member_id) References members(member_id)
+                        );
+
+CREATE TABLE subjects(subject_id int,
+                      subject_nm varchar(90),
+                      Primary key(subject_id)
+                      );
+                    
+CREATE TABLE publishers(publisher_id int Primary Key,
+                        publisher_nm varchar(90)
+                        );
```

Comment:
nope... structure asked for only two columns
![asdasd](https://cloud.githubusercontent.com/assets/13659282/9897102/3eb3eb68-5c62-11e5-9826-e67609cf9e11.png)